src/utils/utils.ts

```diff
@@ -14,4 +15,14 @@ export async function exists(path: string): Promise<boolean> {
 	}
 }
 
+export async function fileExists(uri: vscode.Uri, filename: string): Promise<boolean> {
```

Comment:
Maybe we can use https://code.visualstudio.com/updates/v1_37#_vscodeworkspacefs `vscode-fs`

This works in ssh and wsl scenarios as well.